core/contextmenu_items.js

```diff
@@ -191,7 +191,9 @@ Blockly.ContextMenuItems.addDeletableBlocks_ = function(block, deleteList) {
   if (block.isDeletable()) {
     Array.prototype.push.apply(deleteList, block.getDescendants(false));
   } else {
-    var children = /** @type !Array<!Blockly.BlockSvg> */ (block.getChildren(false));
+    /* eslint-disable indent */
```

Comment:
This disables the rule for the rest of the file. Use `eslint-disable-line` or `eslint-disable-next-line` instead.

https://eslint.org/docs/user-guide/configuring/rules#disabling-rules

Reply:
TIL.  Ta!